source/wp-content/themes/wporg-developer-2023/inc/formatting.php

```diff
@@ -491,7 +491,7 @@ function ( $matches ) {
 							'posts_per_page' => 1,
 						);
 						
-						$query = new WP_Query( $args );
+						$query = new \WP_Query( $args );
```

Comment:
The ones in this file don't seem needed either, as they're also within a class in the default namespace.

Reply:
Reverted